kernel/pte_flags/src/pte_flags_aarch64.rs

```diff
@@ -379,9 +379,9 @@ impl PteFlagsAarch64 {
     /// * Sets the `VALID` bit, as every P4, P3, and P2 entry must be valid.
     #[must_use]
     pub fn adjust_for_higher_level_pte(self) -> Self {
-        self.executable(true)
-            .exclusive(false)
-            .valid(true)
+        Self::empty().accessed(true)
```

Comment:
I understand adding `.accessed(true)` and `.page_descriptor(true)`, but why start with `Self::empty()` and why remove the other parts I had there before? At the very least, we must force `exclusive` to `false`; I don't know about `executable `on aarch64 but I think what I had there is still correct.

Seems like the correct thing is:
```suggestion
        self.executable(true)
            .exclusive(false)
            .accessed(true)
            .page_descriptor(true)
            .valid(true)
```

Reply:
Ok, it works too so I'll push that